ivadomed/config/config_sctTesting.json

```diff
@@ -17,7 +17,9 @@
         },
         "slice_filter_params": {
             "filter_empty_mask": false,
-            "filter_empty_input": true
+            "filter_empty_input": true,
+            "filter_classification" :false,
+             "classifier_path": null
```

Comment:
I was wondering, but not sure, what about creating a new dict which would be similar to `"roi"` --> "slice_wise_classifiier" / applied & path.
At the same time: makes sense to add this filter in this category. Maybe simply move out the "classifier path"?
What are your thoughts @olix86 ?

Reply:
I added it there since the slice filter function needs the classifier path, so if we move it out we would also need to pass the new category to the filter. I don't have a strong opinion in this. either ways is fine with me :)